pkg/resource/aws/aws_route53_zone_ext.go

```diff
@@ -1,5 +1,11 @@
 package aws
 
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+)
+
 func (r *AwsRoute53Zone) String() string {
-	return *r.Name
+	return fmt.Sprintf("%s (%s)", aws.StringValue(r.Name), r.Id)
```

Comment:
Every optional attributs in parenthesis are of the form (AttributeName: AttributeValue). To be more consistent I think we should return `fmt.Sprintf("%s (Id: %s)", aws.StringValue(r.Name), r.Id)`

Reply:
Sure.